src/nodes.coffee

```diff
@@ -2437,6 +2438,10 @@ class IcedRuntime extends Block
     if @isEmpty() then []
     else               super o
 
+  inlineRuntime: (lefthand) ->
+    fs = require('fs')
```

Comment:
My only frustration is that we used to have something that did work here.  So how do you think the browser compiler should work then --- the user has to insert their own runtime?  I'm getting a little confused.